### PR TITLE
Remove MongoDB from supported datastore list

### LINKED ISSUE #684: Support updating process with using transaction on mongoDB cluster
https://github.com/pipe-cd/pipe/blob/1dc024cb30a4d8c557755b98b9693d6e5602d044/pkg/datastore/mongodb/mongodb.go#L140-L143

### LINKED ISSUE #1077: Switch to use statefulset for mongodb and minio
**What would you like to be added**:

**Why is this needed**:

### PR DESCRIPTION
**Pull request by khanhtc1202**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Close #1077
Close #684

**Does this PR introduce a user-facing change?**:

```release-note
Unsupported MongoDB
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -10,7 +10,6 @@ require (
 	github.com/Azure/go-autorest/autorest/adal v0.9.5 // indirect
 	github.com/DataDog/datadog-api-client-go v1.0.0-beta.16
 	github.com/NYTimes/gziphandler v0.0.0-20170623195520-56545f4a5d46
-	github.com/aws/aws-sdk-go v1.36.21 // indirect
```
Comment (nghialv) [inline go.mod:13]: Now we can remove this too. Feel so good.

**Comment by nghialv** [comment]
This makes me feel so good.
Can't wait to merge this. Thank you.
/approve

File: go.mod (hunk 1 above)
Comment (khanhtc1202) [reply go.mod:13]: lol 🤣

**Comment by nakabonne** [comment]
Feel better 😄

**Pull request merged**